packages/frontend-v2/temp/dto-types.ts

```diff
@@ -0,0 +1,218 @@
+import {
```

Comment:
Class-validator and decorators are being weird when importing from `@graduate/common`. To sidestep this blocker, I created a temporary `dto-types` file in the frontend-v2 package so that I can ensure my code works. Fixing class-validator and decorators in the common package will be another ticket.